modules/nf-core/scimap/spatiallda/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "scimap_spatiallda"
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - "YOUR-TOOL-HERE"
```

Comment:
If possible, add a reference to a Conda container for the tool.

Reply:
Unfortunately, there's no conda container for this tool, should we delete the bioconda line together with dependencies?

Reply:
You can try, although I'm not sure how it works with CI tests

Reply:
We deleted the environment.yml